### ISSUE TITLE
Test on the trainset with GT bbox

### ISSUE THREAD

**Issue by mks0601** [facebookresearch/Detectron#666]
Hi FAIR, thanks for sharing your wonderful work.
Can you tell me how to test the pre-trained model in the modelzoo on the train set with gt bbox?

**Comment by ir413** [comment]
Hi @mks0601, I assume that you want to use ground-truth boxes (instead of the boxes predicted by the model) to predict masks or keypoints?

If so, you will need to load the ground-truth boxes (you may find [this](https://github.com/facebookresearch/Detectron/blob/master/detectron/core/test_engine.py#L242-L247) useful) and use them when running the task heads (e.g. see [here](https://github.com/facebookresearch/Detectron/blob/master/detectron/core/test.py#L83) for mask head). Please note that you will need to make sure that the image scale is handled correctly and ensure that the Caffe2 workspace is populated by the features computed by the backbone before running the task heads (you may find [this](https://github.com/facebookresearch/Detectron/blob/master/detectron/core/test.py#L111) function useful).
